src/util.js

```diff
@@ -165,3 +165,12 @@ export function sortCustom(array, order, key) {
       return b - a;
   });
 }
+
+// remove unnecessary preceding 'www.' and etc from url
+export function shortenUrl(url) {
+  const remove = ['http://', 'https://', 'www.'];
```

Comment:
One corner case: what if `www` is not the leading characters, such as this URL: `mywww.com`?

Reply:
I am not sure whether it is a good idea to remove the leading `www`. Technically speaking, `www` is part of the domain name, so `http(s)://www.foobar.com` is not necessarily identical to `http(s)://foobar.com`.

Reply:
This function is meant for display purposes only, not for navigation. It's just supposed to make the url look nicer when displayed, and decrease the horizontal space it takes up. When the user clicks on a url in a dynamic field, the full value including any `http://` or `www` is displayed.

Reply:
I see. But still, you don't want to display `mywww.com` as `my.com`.

Reply:
I could do more advanced filtering, like removing everything before the host, or possibly even using the javascript `[URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL)` functionality to get the relevant bits of information, but I didn't think it was worthwhile to do since our database seems to be fairly clean. Note that that function is only used in one place and on one specific field of a specific query. @dhimmel  thoughts on necessity of this.

Reply:
`URL` seems an overkill. What I meant is something like this:
```
  const targets = ['^http://', '^https://', '^www.'];
  for (const str of targets) {
    regex = new RegExp(str);
    url = url.replace(regex, '');
  }
```
(I renamed `remove` because using verb to name an array seems a little weird.)

Reply:
Modified